Source/HTTP/httpcall.cpp

```diff
@@ -127,7 +127,7 @@ HRESULT perform_http_call(
                         }
                         catch (...)
                         {
-                            HC_TRACE_ERROR(HTTPCLIENT, "HCHttpCallPerform [ID %llu]: failed", static_cast<HC_CALL*>(call)->id);
+                            if (call->traceCall) HC_TRACE_ERROR(HTTPCLIENT, "HCHttpCallPerform [ID %llu]: failed", static_cast<HC_CALL*>(call)->id);
```

Comment:
>HC_TRACE_ERROR [](start = 49, length = 14)

Not wrapping this in braces puts an assumption that this is always going to be a 1-line Macro. Is that guaranteed?

Reply:
Here and elsewhere

---
In reply to: [181256747](https://github.com/Microsoft/libHttpClient/pull/128#discussion_r181256747) [](ancestors = 181256747)